pioinstaller/penv.py

```diff
@@ -12,18 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import json
 import logging
 import os
+import platform
 import subprocess
+import sys
+import time
 
 import requests
 
-from pioinstaller import core, exception, python, util
+from pioinstaller import __version__, core, exception, python, util
 
 log = logging.getLogger(__name__)
 
 
 VIRTUALENV_URL = "https://bootstrap.pypa.io/virtualenv/virtualenv.pyz"
+PORTABLE_PYTHON_64 = (
+    "https://dl.bintray.com/platformio/dl-misc/"
+    "python-portable-windows_amd64-3.7.6.tar.gz"
+)
+PORTABLE_PYTHON_32 = (
+    "https://dl.bintray.com/platformio/dl-misc/"
+    "python-portable-windows_x86-3.7.6.tar.gz"
+)
```

Comment:
Create a dict
```
PORTABLE_PYTHONS = {
"windows_x86": "http://...python-portable-windows_x86-3.7.6.tar.gz",
"windows_amd64": "http://python-portable-windows_amd64-3.7.6.tar.gz"
}
```

Copy from PIO Core `util.get_systype` to your util and use it.